openshift/ci-operator/knative-images/git-init/uidwrapper

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+if ! whoami &> /dev/null; then
+  if [ -w /etc/passwd ]; then
+    echo "${USER_NAME:-default}:x:$(id -u):0:${USER_NAME:-default} user:${HOME}:/sbin/nologin" >> /etc/passwd
+  fi
+fi
+
+[[ -n $@ ]] && exec ${@}
```

Comment:
This feels weird, as it will bypass `.orig` in some cases, and we don't want that :angel: 

```
[[ -x git-init.orig ]] && exec git-init.orig $@ || exec ${@}
```